packages/artillery/lib/platform/aws/iam-cf-templates/aws-iam-lambda-cf-template.json

```diff
@@ -0,0 +1,108 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Description": "CloudFormation template to create an IAM Role for Artillery.io Lambda with attached policies",
+  "Resources": {
+    "ArtilleryDistributedTestingLambdaRole": {
+      "Type": "AWS::IAM::Role",
+      "Properties": {
+        "RoleName": "ArtilleryDistributedTestingLambdaRole",
+        "AssumeRolePolicyDocument": {
+          "Version": "2012-10-17",
+          "Statement": [
+            {
+              "Effect": "Allow",
+              "Principal": {
+                "AWS": "*"
+            },
+              "Action": "sts:AssumeRole",
+              "Condition": {}
+            }
+          ]
+        },
+        "Path": "/",
+        "Policies": [
+          {
+            "PolicyName": "ArtilleryDistributedTestingLambdaPolicy",
+            "PolicyDocument": {
+              "Version": "2012-10-17",
+              "Statement": [
+                {
+                  "Sid": "CreateOrGetLambdaRole",
+                  "Effect": "Allow",
+                  "Action": [
+                    "iam:CreateRole",
+                    "iam:GetRole",
+                    "iam:PassRole",
+                    "iam:AttachRolePolicy"
+                  ],
+                  "Resource": {"Fn::Sub": "arn:aws:iam::${AWS::AccountId}:role/artilleryio-default-lambda-role-*"}
+                },
+                {
+                  "Sid": "CreateLambdaPolicy",
+                  "Effect": "Allow",
+                  "Action": ["iam:CreatePolicy"],
+                  "Resource": {"Fn::Sub": "arn:aws:iam::${AWS::AccountId}:policy/artilleryio-lambda-policy-*"}
+                },
+                {
+                  "Sid": "SQSPermissions",
+                  "Effect": "Allow",
+                  "Action": ["sqs:*"],
+                  "Resource": {"Fn::Sub": "arn:aws:sqs:*:${AWS::AccountId}:artilleryio*"}
+                },
+                {
+                  "Sid": "SQSListQueues",
+                  "Effect": "Allow",
+                  "Action": ["sqs:ListQueues"],
+                  "Resource": "*"
+                },
+                {
+                  "Sid": "LambdaPermissions",
+                  "Effect": "Allow",
+                  "Action": [
+                    "lambda:InvokeFunction",
+                    "lambda:CreateFunction",
+                    "lambda:DeleteFunction",
+                    "lambda:GetFunctionConfiguration"
+                  ],
+                  "Resource": {"Fn::Sub": "arn:aws:lambda:*:${AWS::AccountId}:function:artilleryio-*"}
+                },
+                {
+                  "Sid": "EcrPullImagePermissions",
+                  "Effect": "Allow",
+                  "Action": [
+                    "ecr:GetDownloadUrlForLayer",
+                    "ecr:BatchGetImage"
+                  ],
+                  "Resource": "arn:aws:ecr:*:248481025674:repository/artillery-worker"
```

Comment:
The `EcrPullImagePermissions` needed to be changed so that instead of setting a `Condition` we set the `Resource` to the arn of the repository of the exact account we are pulling from.

The initial policy from the docs was using:
```
{
  "Sid": "EcrPullImagePermissions",
  "Effect": "Allow",
  "Action": [
                    "ecr:GetDownloadUrlForLayer",
                    "ecr:BatchGetImage"
                  ],
  "Resource": "*",
  "Condition": {
     "StringLike": {
          "aws:sourceArn": "arn:aws:lambda:*:123456789000:function:artilleryio-*"
        }
  }
}
```